test/t_end_req.ref

```diff
@@ -33,6 +33,8 @@ SPEC_html_REQ_01
         Globulus shall have 3
         legs:
         
+        
```

Comment:
Why these 2 empty lines ?
This makes the test fail.

Reply:
Yes it pass on my side with a fresh:

```
git clone ssh://git@github.com/HenriGEIST/reqflow.git
cd reqflow
git checkout set_exit_status_on_errors
./configure
make
make check
```

I do not understand why those tow empty line myself.
I just see they are there and on my side it does not pass without them.
This test always failed on my side even before my modifications due to this two blank lines difference.

Reply:
May be a solution is to add -B to the diff test.
This will not explain anything but pass on both of our computers.

Reply:
I suspect a different behaviour of libxml2 when parsing the HTML.
I have libxml2 version 2.9.4. Which is your ?

Reply:
```
dpkg -l | grep libxml2
ii  libxml-libxml-perl                                          2.0134+dfsg-2+b1                     amd64        Perl interface to the libxml2 library
ii  libxml2:amd64                                               2.9.10+dfsg-6.7                      amd64        GNOME XML library
ii  libxml2:i386                                                2.9.10+dfsg-6.7                      i386         GNOME XML library
ii  libxml2-dev:amd64                                           2.9.10+dfsg-6.7                      amd64        Development files for the GNOME XML library
ii  python3-lxml:amd64                                          4.6.3-1                              amd64        pythonic binding for the libxml2 and libxslt libraries
```

Reply:
Thanks.
I tested with libxml2 2.9.12 and I confirm that the root cause is that libxml2 has changed its parsing.
I will merge your pull request and see later how to manage this difference.

Reply:
Thank you !